resources/l10n/fi/LC_MESSAGES/loot.po

```diff
@@ -1,350 +1,350 @@
 msgstr "Pääluettelon pohja ei ole saatavilla"
 
-#: src/gui/qt/helpers.cpp:207
+#: src/gui/qt/helpers.cpp:205
 msgid "Unknown: No revision metadata found"
 msgstr "Tuntematon: version metatietoja ei löydy"
 
-#: src/gui/qt/helpers.cpp:334
+#: src/gui/qt/helpers.cpp:332
 msgid "Invalid regular expression: %1%"
 msgstr "Virheellinen säännöllinen lauseke: %1%"
 
-#: src/gui/qt/main_window.cpp:583
-msgid "Sort Plugins"
-msgstr "Järjestä lisäosat"
+#: src/gui/qt/main_window.cpp:637
+msgid "&Sort Plugins"
+msgstr "&Järjestä lisäosat"
 
-#: src/gui/qt/main_window.cpp:584
-msgid "Update Masterlist"
-msgstr "Päivitä pääluettelo"
+#: src/gui/qt/main_window.cpp:638
+msgid "Update &Masterlist"
+msgstr "&Päivitä pääluettelo"
 
-#: src/gui/qt/main_window.cpp:585
-msgid "Apply Sorted Load Order"
-msgstr "Tallenna uusi järjestys"
+#: src/gui/qt/main_window.cpp:639
+msgid "&Apply Sorted Load Order"
+msgstr "Tallenna uu&si järjestys"
 
-#: src/gui/qt/main_window.cpp:586
-msgid "Discard Sorted Load Order"
-msgstr "Hylkää uusi järjestys"
+#: src/gui/qt/main_window.cpp:640
+msgid "&Discard Sorted Load Order"
+msgstr "&Hylkää uusi järjestys"
 
-#: src/gui/qt/main_window.cpp:589
-msgid "File"
-msgstr "Tiedosto"
+#: src/gui/qt/main_window.cpp:643
+msgid "&File"
+msgstr "&Tiedosto"
 
-#: src/gui/qt/main_window.cpp:590
-msgid "Settings..."
-msgstr "Asetukset..."
+#: src/gui/qt/main_window.cpp:644
+msgid "&Settings..."
+msgstr "&Asetukset..."
 
-#: src/gui/qt/main_window.cpp:591
-msgid "Backup LOOT Data"
-msgstr "Varmuuskopioi LOOTin tiedot"
+#: src/gui/qt/main_window.cpp:645
+msgid "&Backup LOOT Data"
+msgstr "&Varmuuskopioi LOOTin tiedot"
 
-#: src/gui/qt/main_window.cpp:592
-msgid "Open LOOT Data Folder"
-msgstr "Avaa LOOTin tiedostokansio"
+#: src/gui/qt/main_window.cpp:646
+msgid "&Open LOOT Data Folder"
+msgstr "Avaa &LOOTin tiedostokansio"
 
-#: src/gui/qt/main_window.cpp:593
-msgid "Quit"
-msgstr "Poistu"
+#: src/gui/qt/main_window.cpp:647
+msgid "&Quit"
+msgstr "&Poistu"
 
-#: src/gui/qt/main_window.cpp:595
-msgid "Game"
-msgstr "Peli"
+#: src/gui/qt/main_window.cpp:649
+msgid "&Game"
+msgstr "P&eli"
 
-#: src/gui/qt/main_window.cpp:596
-msgid "Edit Groups..."
-msgstr "Muokkaa ryhmiä..."
+#: src/gui/qt/main_window.cpp:650
+msgid "&Edit Groups..."
+msgstr "&Muokkaa ryhmiä..."
 
-#: src/gui/qt/main_window.cpp:597
-msgid "Search Cards..."
-msgstr "Hae korteista..."
+#: src/gui/qt/main_window.cpp:651
+msgid "Searc&h Cards..."
+msgstr "&Hae korteista..."
 
-#: src/gui/qt/main_window.cpp:598
-msgid "Copy Load Order"
-msgstr "Kopioi järjestys"
+#: src/gui/qt/main_window.cpp:652
+msgid "Copy &Load Order"
+msgstr "Kopioi &järjestys"
 
-#: src/gui/qt/main_window.cpp:599
-msgid "Copy Content"
-msgstr "Kopioi sisältö"
+#: src/gui/qt/main_window.cpp:653
+msgid "&Copy Content"
+msgstr "&Kopioi sisältö"
 
-#: src/gui/qt/main_window.cpp:600
-msgid "Refresh Content"
-msgstr "Päivitä sisältö"
+#: src/gui/qt/main_window.cpp:654
+msgid "&Refresh Content"
+msgstr "Päivitä &sisältö"
```

Comment:
I think this and the `Tallenna uu&si järjestys` mnemonic are ambiguous as they both appear in the Game menu.